centml/cli/cluster.py

```diff
@@ -21,6 +21,22 @@ def convert(self, value, param, ctx):
             return None  # to avoid warning from lint for inconsistent return statements
 
 
+# Hardware pricing tier that loads choices dynamically
+class HardwarePricingTier(click.ParamType):
+    name = "choice"
```

Comment:
if this name is used somewhere, can we give it a more descriptive name?

Reply:
fixed